plugins/kubernetes/app/models/kubernetes/deploy_executor.rb

```diff
@@ -70,7 +70,8 @@ def wait_for_resources_to_complete(release_docs, timeout)
 
       loop do
         statuses = resource_statuses(release_docs)
-        if statuses.select { |s| s.kind == "Pod" }.none?
+        interesting = statuses.select { |s| s.kind == "Pod" || !s.live } # ignore boring things that rarely fail
```

Comment:
removed a theoretical bug here were failed deployments without a pod would be declared success